jobs/quicksuggest2bq/quicksuggest2bq/main.py

```diff
@@ -23,34 +23,34 @@ class KintoSuggestion:
     # if there's new unexpected fields, ensuring we take the
     # appropriate actions to update the table schemas.
     advertiser: str
-    click_url: str
     iab_category: str
     icon: str
-    impression_url: str
     id: int
     keywords: List[str]
     title: str
     url: str
+    click_url: Optional[str] = None
```

Comment:
Our in-house Wikipedia suggestions don't have `click_url` and `impression_url` unlike the adM-provided counterparts.

Reply:
Might be worth adding a comment about it above this line, but I'll leave it to your judgement.